source/world_builder/features/subducting_plate_models/temperature/mass_conserving.cc

```diff
@@ -118,12 +118,11 @@ namespace WorldBuilder
           prm.declare_entry("density", Types::Double(3300),
                             "The reference density of the subducting plate in $kg/m^3$");
 
-          prm.declare_entry("plate velocity", Types::OneOf(Types::Double(0.05),Types::Array(Types::ValueAtPoints(0.05, std::numeric_limits<uint64_t>::max()))),
-                            "The velocity with which the plate subducts in meters per year. Default is 5 cm/yr");
+          prm.declare_entry("spreading velocity", Types::OneOf(Types::Double(0.05),Types::Array(Types::ValueAtPoints(0.05, std::numeric_limits<uint64_t>::max()))),
+                            "The velocity with which the ridge spreads and create the plate in meters per year. Default is 5 cm/yr");
 
-          prm.declare_entry("subducting velocity", Types::OneOf(Types::Double(-1), Types::Array(Types::Array(Types::Double(-1), 1), 1)),
-                            "The velocity with which the ridge is moving through time, and how long the ridge "
-                            "has been moving. First value is the velocity, second is the time. Default is [0 cm/yr, 0 yr]");
+          prm.declare_entry("subducting velocity", Types::OneOf(Types::Double(0.05), Types::Array(Types::Array(Types::Double(0.05), 1), 1)),
+                            "The velocity with which the slab is subducting through time. Default is 5 cm/yr");
```

Comment:
I don't entirely remember the discussion. We decided to make both a required variable, instead of giving one a special value option which means it is the same as the other (e.g. "subducting velocity":-1 means "subducting velocity"="ridge velocity")? See question on comment below.

Reply:
I added the subducting velocity to the required parameters above.